pom.xml

```diff
@@ -61,6 +61,10 @@
     <maven-jar-plugin.version>3.2.2</maven-jar-plugin.version>
     <maven-project-info-reports-plugin.version>3.3.0</maven-project-info-reports-plugin.version>
     <maven-remote-resources-plugin.version>1.7.0</maven-remote-resources-plugin.version>
+    <os-maven-plugin.version>1.4.1.Final</os-maven-plugin.version>
+    <protobuf-maven-plugin.version>0.6.1</protobuf-maven-plugin.version>
+    <protoc3-maven-plugin.version>3.19.2</protoc3-maven-plugin.version>
+    <protoc-gen-grpc-java-plugin.version>1.45.1</protoc-gen-grpc-java-plugin.version>
```

Comment:
It might be better to define a `<grpc.version>1.46.0</grpc.version>`, and then point it to `grpc.version`